web/core/store/user/user-membership.store.ts

```diff
@@ -1,7 +1,7 @@
-import { set } from "lodash";
+import { set, update } from "lodash";
```

Comment:
Import set and update separately.
https://stackoverflow.com/questions/35250500/correct-way-to-import-lodash

Reply:
Resolved